radlab-ui/webapp/src/pages/api/github/index.ts

```diff
@@ -0,0 +1,33 @@
+import { getGitHubModulesData, getSecretKeyValue } from "@/utils/api"
+import { envOrFail } from "@/utils/env"
+import { withAuth } from "@/utils/middleware"
+import { AuthedNextApiHandler } from "@/utils/types"
+import { NextApiResponse } from "next"
+
+const GIT_TOKEN_SECRET_KEY_NAME = envOrFail(
+  "GIT_TOKEN_SECRET_KEY_NAME",
+  process.env.GIT_TOKEN_SECRET_KEY_NAME,
+)
```

Comment:
Will the whole app fail to start if this is not provided? Is this optional (as in we by default pull from GoogleCloudPlatform/radlab)? If so, can we move the `envOrFail` into the handler itself (`getModuleDataFromGithub`).

Reply:
This block of code can be moved inside  `handler`  function since app will not fail to start if not provided.
Code modified accordingly.